google/cloud/bigquery/v2/minimal/internal/job_logging.cc

```diff
@@ -57,6 +57,33 @@ StatusOr<GetJobResponse> BigQueryJobLogging::GetJob(
   return response;
 }
 
+StatusOr<ListJobsResponse> BigQueryJobLogging::ListJobs(
+    rest_internal::RestContext& rest_context, ListJobsRequest const& request) {
+  char const* context = __func__;
+  GCP_LOG(DEBUG) << context
```

Comment:
if you provide `operator<<(std::ostream&, ListJobsRequest const&)` then this can be simplified using  one of the [log wrappers](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/internal/log_wrapper.h).  The code would become:

```cc
StatusOr<ListJobsResponse> BigQueryJobLogging::ListJobs(
    rest_internal::RestContext& rest_context, ListJobsRequest const& request) {
  return google::cloud::internal::LogWrapper(
      [this](auto& context, auto const& request) { return child_->ListJobs(context, request); },
      rest_context, request, __func__, tracing_options_);
}
```